### DISCUSSION TITLE
issue in flux store multivendor flutter code

### DISCUSSION THREAD

**Discussion by anand-knowminal** [inspireui/support#5188] [Working on Features]
Hi,
We purchased single licence fluxstore multi vendor flutter code. 
Downloaded and opened the project in visual studio code. when we run **flutter pub get**, it is throwing  **version solving failed** issue.

Could you please help us to rectify this issue? Your guidance will be much appreciated. 

Development environment flutter version is  **Flutter version 2.3.0-12.1.pre**

**Comment by vero-pix** [comment]
same issue hereBecause every version of flutter_test from sdk depends on async 2.6.1 and fstore depends on async 2.5.0, flutter_test from sdk is forbidden.
So, because fstore depends on flutter_test any from sdk, version solving failed.
pub get failed (1; So, because fstore depends on flutter_test any from sdk, version solving failed.)
P

**Comment by sachin-dahal** [comment]
It might be because of not updating the packages to support the latest version of the Flutter/Dart SDK version. The developer must upgrade the packages accordingly to support the latest version of the current SDK. Now, in this case, you might not be able to do much other than downgrading your current SDK version to the particular version that all the current package in usage are compatible with.

Other solutions here could be (hope it'll work)... 
- Cloning/copying the project again and just upgrading each package manually that the latest Flutter version supports.
- Upgrading your flutter version to the dev channel
```	
        $ flutter channel dev
	$ flutter upgrade
```
:blush:

**Reply by minhcasi** [reply]
Hi @all

It is required to use the FluxStore with Stable Channel Flutter. Fluxstore 2.2 is compatible with Flutter 2.2, to upgrade it work with dev or master channel, you need to update the pubspec.yaml library

Ref changelog: https://www.notion.so/inspireui/Announcing-FluxStore-2-2-x-0310c331c37c4d3a92a775377de7eb73

This GitHub is only use for new feature request, please post your ticket support to http://support.inspireui.com/
